forge-gui/res/cardsfolder/upcoming/cruel_somnophage_cant_wake_up.txt

```diff
@@ -0,0 +1,18 @@
+Name:Cruel Somnophage
+ManaCost:1 B
+Types:Creature Nightmare
+PT:*/*
+S:Mode$ Continuous | EffectZone$ All | CharacteristicDefining$ True | SetPower$ X | SetToughness$ X | Description$ CARDNAME's power and toughness are each equal to the number of creature cards in all graveyards.
+SVar:X:Count$TypeInAllYards.Creature
+DeckHints:Ability$Mill|Discard|Graveyard
+DeckHas:Ability$Mill
+AlternateMode:Adventure
+Oracle:Cruel Somnophage's power and toughness are each equal to the number of creature cards in all graveyards.
+
+ALTERNATE
+
+Name:Can't Wake Up
+ManaCost:1 U U
+Types:Sorcery Adventure
+A:SP$ Mill | NumCards$ 4 | ValidTgts$ Player | TgtPrompt$ Choose a player | SpellDescription$ Target player mills four cards. (Then exile this card. You may cast the creature later from exile.)
```

Comment:
I'd remove this TgtPrompt... never like it when the prompt doesn't include the word "target" anyway